packages/devtools-launchpad/src/tools/mc/make-bundle.js

```diff
@@ -13,7 +13,7 @@ function makeBundle({ outputPath, projectPath, watch = false, updateAssets = fal
   const webpackConfig = require(path.resolve(projectPath, "webpack.config.js"));
 
   return new Promise((resolve, reject) => {
-    if (updateAssets) {
+    if (!updateAssets) {
```

Comment:
got the negation wrong. If we're not updating the assets we should delete the records path...